frontend/apps/marketing/src/themes/code.org/index.ts

```diff
@@ -13,11 +13,48 @@ const theme = createTheme({
       styleOverrides: {
         root: {
           ['&.MuiButton-contained.MuiButton-colorPrimary']: {
-            backgroundColor: '#8c52ba',
+            backgroundColor: 'var(--brand-purple-50)',
```

Comment:
We can use our existing css variables here. Not sure if this is a good long term solution, but for right now it's pretty nice!

Reply:
This is OK until we add the theme palette